frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -272,6 +273,11 @@ class Receive extends Component<Props, State> {
                                         title={getScriptName(scriptType)}>
                                         {t(`receive.scriptType.${scriptType}`)}
                                     </Radio>
+                                    {scriptType === 'p2tr' && this.availableScriptTypes.indexOf('p2tr') === addressDialog.addressType && (
```

Comment:
why not the same condition for activeness as above in `checked=...`? `addressDialog.addressType === i`?

` this.availableScriptTypes.indexOf('p2tr')` is the same as `i` as we are mapping over it.

Reply:
🙈   Fixed and squashed